po/mmex.pot

```diff
@@ -54,6 +54,18 @@ msgstr ""
 msgid "Enable to help us sending anonymous data about MMEX usage."
 msgstr ""
 
+#: util.cpp:1495
+msgid "System Locale: %s"
+msgstr ""
+
+#: util.cpp:1496
+msgid "User Interface Language: %s"
+msgstr ""
+
+#: util.cpp:1505
+msgid "%ix%i %i-bit %ix%ippi\n"
```

Comment:
Remove newline?

From:
`msgid "%ix%i %i-bit %ix%ippi\n"`
to:
`msgid "%ix%i %i-bit %ix%ippi"`